### DISCUSSION TITLE
How to generate different types of fluid particles in the same place？

### DISCUSSION THREAD

**Discussion by csusu953** [DualSPHysics/DualSPHysics#265] [DUALSPHYSICS_V5.4_BETA]
Dear Developers，
     I would like to know if it is possible to generate different types of particles at the same spatial location during initialization through an XML file, and how to achieve this?
     For example, in the multiphase flow case, I defined two types of fluid particles with different viscosities (marked MK=1 and MK=2), and I hope to make the spatial positions of these two types of particles overlap during initialization. In the XML file, I used the same geometric modeling parameters for the two types of fluid particles. However, the program would terminate and indicate that the fluid particles were not successfully generated.
     Looking forward to your reply, thank you very much!

**Comment by alexbexe** [comment]
You can not get any overlap of the particles.
During the pre-processing the lattice used for GenCase will create only one type of particles in the nodes of that lattice.

Regards

**Reply by csusu953** [reply]
Thank you very much for your response!
I would like to know whether this limitation exists only in the **GenCase** program or also in the **DualSPHysics** ? 
In other words, is it possible to achieve particle overlap by inputting a bi4 file externally (instead of through GenCase)? Or is particle overlap also disabled in **DualSPHysics** because it affects steps like CLL and particle searching?
